README.md

```diff
@@ -108,18 +109,18 @@ Here is an overview of them, combined with ratings on properties that guided the
 
 Please take this table with a grain of salt: a different crate might very well be more suitable for your specific use case.
 
-| Library            | Status        | Lin | Mac | Win | Base | User | Proj | Conv |
-| ------------------ | ------------- | :-: | :-: | :-: | :--: | :--: | :--: | :--: |
-| `app_dirs`         | Unmaintained  |  ✔  |  ✔  |  ✔  |  🞈   |  ✖   |  ✔   |  ✖   |
-| `app_dirs2`        | Maintained    |  ✔  |  ✔  |  ✔  |  🞈   |  ✖   |  ✔   |  ✖   |
-| **dirs-next**      | **Developed** |  ✔  |  ✔  |  ✔  |  ✔   |  ✔   |  ✖   |  ✔   |
-| dirs               | Unmaintained  |  ✔  |  ✔  |  ✔  |  ✔   |  ✔   |  ✖   |  ✔   |
-| `directories-next` | Developed     |  ✔  |  ✔  |  ✔  |  ✔   |  ✔   |  ✔   |  ✔   |
-| `s_app_dir`        | Unmaintained? |  ✔  |  ✖  |  🞈  |  ✖   |  ✖   |  🞈   |  ✖   |
-| `standard_paths`   | Maintained    |  ✔  |  ✖  |  ✔  |  ✔   |  ✔   |  ✔   |  ✖   |
-| `xdg`              | Maintained    |  ✔  |  ✖  |  ✖  |  ✔   |  ✖   |  ✔   |  🞈   |
-| `xdg-basedir`      | Unmaintained? |  ✔  |  ✖  |  ✖  |  ✔   |  ✖   |  ✖   |  🞈   |
-| `xdg-rs`           | Obsolete      |  ✔  |  ✖  |  ✖  |  ✔   |  ✖   |  ✖   |  🞈   |
+| Library                                                         | Status        | Lin | Mac | Win | Base | User | Proj | Conv |
+| --------------------------------------------------------------- | ------------- | :-: | :-: | :-: | :--: | :--: | :--: | :--: |
```

Comment:
There were links to other crates there. I removed those to keep the simple.
People could go to crates.io and enter these crate names.
Also long lines are hard to work in vim.

Reply:
Sorry, didn't realize `dirs-rs` originally had links to the other crates in the comparison. I personally like the links for the ease of looking and comparing other possible information for the crates.